script/tool/lib/src/common/package_command.dart

```diff
@@ -563,23 +563,13 @@ abstract class PackageCommand extends Command<void> {
     await for (final PackageEnumerationEntry package
         in getTargetPackages(filterExcluded: filterExcluded)) {
       yield package;
-      yield* getSubpackages(package.package).map(
-          (RepositoryPackage subPackage) =>
-              PackageEnumerationEntry(subPackage, excluded: package.excluded));
+      yield* Stream<PackageEnumerationEntry>.fromIterable(package.package
+          .getSubpackages()
+          .map((RepositoryPackage subPackage) =>
+              PackageEnumerationEntry(subPackage, excluded: package.excluded)));
     }
   }
 
-  /// Returns all Dart package folders (e.g., examples) under the given package.
-  Stream<RepositoryPackage> getSubpackages(RepositoryPackage package,
```

Comment:
This is just drive-by refactoring; this helper being here predates the creation of `RepositoryPackage`, and that class is a better home for this method (especially since it already has a similar method to find all example apps).

It changed its return from `Stream` to `Iterable` to match the existing helper function, and because we don't actually benefit from using `Stream`.